assets/app/views/project.html

```diff
@@ -7,11 +7,20 @@
       </div>
       <!-- TODO handle things that don't live under services -->
       <alerts alerts="alerts"></alerts>
-      <div ng-if="(services | hashSize) == 0">
-        <div>
-          <em>{{emptyMessage}}</em>
-        </div>
+      <div ng-if="loadingServices">Loading...</div>
```

Comment:
we don't want to show the "Your project is empty" message if they filtered down project contents to nothing.
Keep this structure:
```
<div ng-if="(services | hashSize) == 0">
  <div ng-if="showGetStarted">
  </div>
  <div ng-if="!showGetStarted">
    <em>{{emptyMessage}}</em>
  </div>
</div>
```